packages/server/src/fhir/operations/utils/parameters.ts

```diff
@@ -60,6 +60,48 @@ export function parseInputParameters<T>(operation: OperationDefinition, req: Req
   }
 }
 
+function parseQueryString(
+  query: Record<string, any>,
+  inputParams: OperationDefinitionParameter[]
+): Record<string, any> {
+  const parsed = Object.create(null);
+  for (const param of inputParams) {
+    const value = query[param.name];
+    if (!value) {
+      continue;
+    }
+    if (param.type?.match(/^[A-Z]/)) {
+      // Query parameters cannot contain complex types
+      throw new OperationOutcomeError(
+        badRequest(`Complex parameters (${value} - ${param.type}) cannot be passed via query string`)
+      );
+    }
+
+    switch (param.type) {
+      case 'integer':
+      case 'positiveInt':
+      case 'unsignedInt':
+        parsed[param.name] = parseInt(value, 10);
+        break;
+      case 'decimal':
+        parsed[param.name] = parseFloat(value);
+        break;
+      case 'boolean':
+        if (value === 'true') {
```

Comment:
Super minor, but I suspect this pattern is repeated enough that we could benefit from a `parseBoolean` method in `@medplum/core`.